### PR TITLE
Commander : commander is  not case sensitive now.

### PR DESCRIPTION
**Pull request by SushantChandla**

### CODE REVIEW

**Comment by l7ssha** [comment]
Why would you want case sensitive commands? Why that should be marged? If that should be merged maybe better would be to provide option to enable/disable this feature?

**Comment by SushantChandla** [comment]
> Why would you want case sensitive commands?

I don't want it to be case sensitive.
It was case sensitive, so i have fixed that.

File: nyxx_commander/lib/src/CommandHandler.dart [hunk 1]
```diff
@@ -19,7 +19,7 @@ abstract class CommandEntity {
   CommandEntity? get parent;
 
   /// A list of valid command names
-  List<String> get commandNames => [if (this.name.isNotEmpty) this.name, ...this.aliases];
+  List<String> get commandNames => [if (this.name.isNotEmpty) this.name.toLowerCase(),]+aliases.map((e) => e.toLowerCase()).toList();
```
Comment (l7ssha) [inline nyxx_commander/lib/src/CommandHandler.dart:22]: `...aliases.map((e) => e.toLowerCase()).toList()` would be better, but i'm not sure if that's possible

Comment (SushantChandla) [reply nyxx_commander/lib/src/CommandHandler.dart]: Updated.

**Review by HarryET — APPROVED** [review]
Looks good :)

**Pull request merged**